src/components/sidebar/types.ts

```diff
@@ -44,4 +40,34 @@ export interface SidebarProps {
    * Text to be shown as the title of the sidebar.
    */
   title: string
+
+  /**
+   * @TODO - write description
```

Comment:
Non-blocking: we could drop in the description from `TutorialsSidebar`:

```suggestion
   * Optional. If true, the title of the sidebar will be visually hidden.
```

Reply:
Updated in https://github.com/hashicorp/dev-portal/pull/335/commits/fc75a3ec2ff11a7c1d458964440d6c940207074f!